kolibri/plugins/learn/assets/src/views/CardList.vue

```diff
@@ -321,4 +316,10 @@
     margin-bottom: $footer-height;
   }
 
+  @media only screen and (max-width: 720px) {
```

Comment:
[Blocking] This indeed reduces some space. However, looking at our designs, it seems that we want to keep some space between the text area and the progress bar. We need to only address unwanted or unexpected spaces. Therefore this needs to be reverted. Thank you for your effort.